test/rabbitmanager_test.js

```diff
@@ -159,10 +159,11 @@ suite('Rabbit Wrapper', function() {
 
     assert(queues instanceof Array);
     assert(queues.length > 0);
-    assert(_.has(queues[0], 'memory'));
-    assert(_.has(queues[0], 'messages'));
-    assert(_.has(queues[0], 'messages_details'));
-    assert(_.has(queues[0], 'messages_ready'));
+    // TODO: the rabbit server doesn't return these keys, it seems
```

Comment:
Hi @djmitche ! Saw this patch and just wanted to chime in to provide some insights based on my experience.

It's very likely we run into serious version dependency issues here. I found rabbitmq-management had rather unstable/under-documented HTTP APIs. Different versions could have very different query points and responses. IIRC, the development was done with the then-latest rabbitmq (~3.6.5) and these tests used to pass on that version. Though not written by me, I believe RabbitMonitor in particular makes heavy use of `messages_details`, which contains useful information for monitoring (https://cdn.rawgit.com/rabbitmq/rabbitmq-management/rabbitmq_v3_6_8/priv/www/doc/stats.html).

I couldn't find changelog or any compatibility guarantees (even version schemas) for the management HTTP API. In retrospect, it was actually an important issue that we failed to pay enough attention to. I'd suggest a careful review on the versions of rabbitmq _and_ rabbitmq-management plugin (figuring out the target version and unifying all development/testing/production infra to that version, etc.) before moving on.

Hope that would help!